content/research_report/introductions.md

```diff
@@ -1,61 +1,72 @@
 __Non-grey matter tissue signal__ (such as _white matter_ and _cerebrospinal fluid_), unlikely to reflect neuronal activity {cite:p}`fox_pnas_2005`,
 is captured by averaging signal within anatomically-derived masks.
 __Global signal__ is calculated by averaging signal within the _full brain_ mask {cite:p}`fox_pnas_2005`.
+These three classes of regressors can be expanded to their first temporal derivatives and their quadratic terms {cite:p}`satterthwaite_2013`.
+__Scrubbing__ {cite:p}`power_scrubbing_2012` is a volume censoring approach to remove high motion segments in which the framewise displacement 
+(see section {ref}`framewise-displacement`)
+exceeds some threshold. 
+The scrubbing approach is applied along side head motion parameters and tissue signal regressors.
+__Temporal high-pass filtering__ accounts for low-frequency signal drifts introduced by _physiological and scanner noise sources_.
+Aside from regressors directly modeling noise derived from realignment measures or anatomical properties,
+other approaches capture the impact of motion and non-neuronal physiological activity through data-driven methods. 
+The principle component based method __CompCor__ {cite:p}`behzadi_compcor_2007,muschelli_compcor_2014` extracts principal components from white matter and cerebrospinal fluid masks to estimate non-neuronal activity.
+Independent component analysis based methods estimate spatial independent components representing brain activity and/or noise, 
+and then identify the components related to head-motion through using a data-driven classifier (__ICA-FIX__ {cite:p}`salimi-khorshidi_automatic_2014`)
+or a pre-trained model (__ICA-AROMA__ {cite:p}`aroma`).
 
-These four classes of regressors can be expanded to their first temporal derivatives and their quadratic terms {cite:p}`satterthwaite_2013`.
-The principle component based method __CompCor__ {cite:p}`behzadi_compcor_2007` extracts principal components from white matter and cerebrospinal fluid masks to estimate non-neuronal activity.
-Independent component analysis based methods, __ICA-FIX__ {cite:p}`salimi-khorshidi_automatic_2014` and __ICA-AROMA__ {cite:p}`aroma`,
-estimate independent component time series related to head-motion through a data-driven classifier (ICA-FIX) or a pre-trained model (ICA-AROMA).
-Different strategies have particular strengths,
-and an optimal approach often involves combining a few classes of regressors described above.
+A denoising approach often involves combining a few classes of regressors described above.
+Different strategies have particular strengths and limitations, and researchers can make their own choices based on properties of their data.
+Head motion combied with non-grey matter tissue signal is one of the most basic approach {cite:p}`fox_pnas_2005`. 
+To gain the best denosing results, it often requires the their expansions to module non-linear impact of noise. 
```

Comment:
Suggestion (if I have understood correctly): Move this and combine with line 31. e.g. These three classes of regressors can be expanded to their first temporal derivatives and their quadratic terms {cite:p}`satterthwaite_2013` to model the impact of non-linear noise, often required for optimal denoising results.